packages/salesforcedx-vscode-core/src/commands/forceRefreshSObjects.ts

```diff
@@ -232,46 +237,17 @@ function getStandardSObjectsDirectory(projectPath: string) {
 }
 
 export async function checkSObjectsAndRefresh(projectPath: string) {
-  const hasDefaultUsernameSet =
-    (await workspaceContext.getConnection()).getUsername() !== undefined;
-  if (projectPath && hasDefaultUsernameSet) {
-    if (!fs.existsSync(getStandardSObjectsDirectory(projectPath))) {
-      telemetryService.sendEventData(
-        'sObjectRefreshNotification',
-        { type: 'No SObjects' },
-        undefined
-      );
-      const message = nls.localize('sobjects_refresh_needed');
-      const buttonTxt = nls.localize('sobjects_refresh_now');
-      const shouldRefreshNow = await notificationService.showInformationMessage(
-        message,
-        buttonTxt
-      );
-      if (shouldRefreshNow && shouldRefreshNow === buttonTxt) {
-        telemetryService.sendEventData(
-          'sObjectRefreshNotification',
-          { type: 'Requested Refresh' },
-          undefined
-        );
-        try {
-          await forceRefreshSObjects(SObjectRefreshSource.StartupMin);
-        } catch (e) {
-          telemetryService.sendException(e.name, e.message);
-          throw e;
-        }
-      } else {
-        telemetryService.sendEventData(
-          'sObjectRefreshNotification',
-          { type: 'Refresh Request Cancelled' },
-          undefined
-        );
-      }
-    } else {
-      telemetryService.sendEventData(
-        'sObjectRefreshNotification',
-        { type: 'SObjects exist' },
-        undefined
-      );
+  if (projectPath && !fs.existsSync(getStandardSObjectsDirectory(projectPath))) {
+    telemetryService.sendEventData(
+      'Refresh SObjects',
```

Comment:
I think we had a conversation on a PR some time ago about starting to use a naming convention for telemetry events. I don't recall what we decided on, @sfsholden, @AnanyaJha  or @xyc do you remember if we decided on a naming convention ?

Reply:
I am not sure - I do not think I was a part of this discussion

Reply:
Since no one remembers let's just keep the telemetry name to what it used to be (`sObjectRefreshNotification`) so  it is still usable in current dashboards and queries.